pom.xml

```diff
@@ -584,5 +583,78 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Make sure the source assembly has the right name (includes "incubating") -->
+        <profile>
+            <id>apache-release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-assembly-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>source-release-assembly</id>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>single</goal>
+                                </goals>
+                                <!-- heads up: combine.self in the following is highlighted
+                                    as an error in Eclipse's xml editor view.
+                                    Just ignore that.
+                                    See  https://issues.apache.org/jira/browse/MNG-5454  sigh.
+                                 -->
+                                <configuration combine.self="append">
+                                    <finalName>apache-iotdb-${project.version}-incubating</finalName>
```

Comment:
Oh, now I get what Justin means... just add "incubating" here is fine..

Reply:
yeah.. therefore I changed the name of the module back to "iotdb-parent" which is more standard : )